test/BasicAccessTest.php

```diff
@@ -109,7 +107,7 @@ public function testIsAuthenticatedWithValidCredential()
 
         $user = $basicAccess->authenticate($this->request->reveal());
         $this->assertInstanceOf(UserInterface::class, $user);
-        $this->assertEquals('Aladdin', $user->getIdentity());
+        $this->assertEquals($username, $user->getIdentity());
```

Comment:
I think this assertion is only testing the mock and therefor worthless. Should i remove it?

Reply:
Yes, please.